update marina doc + add ionio-example (#18)
vulpemventures/docs.vulpem.com: docs/marina/getting-started.md

You can review the full API for that object [here](api.md). 

## Install marina-provider

> To make your life easier, we have created a package that provides utilities and common types related to Marina. 

Reviewer: maybe better?
```suggestion
To make things easier, we have created a package that provides utilities and common types related to Marina. 
```

  console.log('Please install Marina extension!');
}
```
You can review the full API for the `window.marina` object [here](api.md).

## Connect to outside world

> Marina is a Liquid wallet running in your browser. It connects to an [ElectrumX](https://electrumx.readthedocs.io/) Liquid instance. Marina is using the WebSocket protocol to connect to the ElectrumX server. Marina use the Blockstream's ElectrumX server by default. You can change the endpoint in the settings/explorer tab of your extension.

Reviewer: ```suggestion
Marina connects to Blockstream's [ElectrumX](https://electrumx.readthedocs.io/) server via WebSocket by default. You can customize the WS endpoint in the settings/explorer tab of the extension.
```
